MonoGame.Framework/SDL/SDLGamePlatform.cs

```diff
@@ -125,6 +125,7 @@ private void SdlRunLoop()
 
                     if (!_keys.Contains(key))
                         _keys.Add(key);
+                    _view.CallTextInput ((char)ev.Key.Keysym.Sym, key);
```

Comment:
What keeps you from getting two events for the same key press?  One from this `Sdl.EventType.KeyDown` and one from `Sdl.EventType.TextInput` below?